ui/apps/ui/src/app/pages/search-page/search-page.component.html

```diff
@@ -68,10 +68,43 @@
           ></ess-results-with-pagination>
         </div>
         <div
-          class="col-sm-3 col-12 right-column"
-          style="border-left: 1px solid #d9dee2; padding-left: 20px"
+          class="col-sm-3 col-12 right-column pl-0"
         >
           <ess-articles></ess-articles>
+
+          <div class="right-links-wrapper">
+            <h4>Articles</h4>
+
+            <div class="link-box">
+              <a href="https://beta.knowledge-hub.eosc-portal.eu/articles/article_01.html">
```

Comment:
the same question. I assume that currently there is no production version of that site, but shouldn't it be dynamically assigned?